internal/terraform/aws/nat_gateway_test.go

```diff
@@ -0,0 +1,40 @@
+package aws_test
+
+import (
+	"infracost/internal/testutil"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestNatGatewayIntegration(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in short mode")
+	}
+
+	tf := `
+resource "aws_nat_gateway" "nat" {
+	allocation_id = "fake_eip_id"
+	subnet_id     = "fake_subnet_id"
```

Comment:
Should we try to stick with the cloud provider's ID format for these sorts of things? I suppose it doesn't matters as Terraform doesn't validate them and if they start to do so, we can search/replace...